gravitee-apim-console-webui/src/components/notifications/notifications-list/notifications-add-notification-dialog/notifications-add-notification-dialog.component.ts

```diff
@@ -20,34 +20,42 @@ import { FormControl, FormGroup, Validators } from '@angular/forms';
 import { Notifier } from '../../../../entities/notification/notifier';
 import { NewNotificationSettings } from '../../../../entities/notification/newNotificationSettings';
 import { NotificationSettings } from '../../../../entities/notification/notificationSettings';
-import { UIRouterStateParams } from '../../../../ajs-upgraded-providers';
+
+type Reference = {
+  referenceType: 'API' | 'APPLICATION' | 'PORTAL';
+  referenceId: string;
+};
 
 export interface NotificationsAddNotificationDialogData {
   notifier: Notifier[];
+  reference: Reference;
 }
 
 export type NotificationsAddNotificationDialogResult = NewNotificationSettings;
 
 @Component({
-  selector: 'notifications-add-notification-dialog',
+  selector: 'notification-settings-add-notification-dialog',
   template: require('./notifications-add-notification-dialog.component.html'),
   styles: [require('./notifications-add-notification-dialog.component.scss')],
 })
 export class NotificationsAddNotificationDialogComponent {
```

Comment:
```suggestion
export class NotificationSettingsAddDialogComponent {
```

Reply:
Done

Reply:
you sure.? i stil can see NotificationsAddNotificationDialogComponent ant not `NotificationSettingsAddDialogComponent `

Reply:
Done